sql/plsql/PlSqlParser.g4

```diff
@@ -4392,7 +4392,21 @@ table_alias
     ;
 
 where_clause
-    : WHERE (CURRENT OF cursor_name | expression)
+    : WHERE ((CURRENT OF cursor_name | expression) | all_some_any_where_statment)
+    ;
+
+all_some_any_where_statment
+    : expression dingliang_compare_operator (SOME | ALL | ANY) '(' expression (',' expression)* ')'
+    ;
+
+//定量比较运算符
+dingliang_compare_operator
```

Comment:
I would use the English word for `dingliang` here. According to Google, it's `quantitative`.